From 4ff5f49c0c7457705e23751b277c4b521d4bede3 Mon Sep 17 00:00:00 2001
From: Oscar Gomez Fuente <oscargomez@tedesys.com>
Date: Mon, 4 Apr 2016 17:43:58 +0200
Subject: [PATCH] usbfs memory mb point grey image size bigger 2mb

---
 drivers/usb/core/devio.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/core/devio.c b/drivers/usb/core/devio.c
index 986abde..895932e 100644
--- a/drivers/usb/core/devio.c
+++ b/drivers/usb/core/devio.c
@@ -113,7 +113,7 @@ enum snoop_when {
 #define USB_DEVICE_DEV		MKDEV(USB_DEVICE_MAJOR, 0)
 
 /* Limit on the total amount of memory we can allocate for transfers */
-static unsigned usbfs_memory_mb = 16;
+static unsigned usbfs_memory_mb = 256;
 module_param(usbfs_memory_mb, uint, 0644);
 MODULE_PARM_DESC(usbfs_memory_mb,
 		"maximum MB allowed for usbfs buffers (0 = no limit)");
-- 
1.9.1

